src/block/carousel/style.js

```diff
@@ -169,7 +169,7 @@ const Styles = props => {
 			/>
 			<BlockCss
 				{ ...propsToPass }
-				selector=".stk-block-carousel__dots"
+				selector=".%s > .stk-block-carousel__content-wrapper > .stk-block-carousel__dots"
```

Comment:
`>` is not allowed for saved styles because this will produce an error in multisite. Please revert, let's not encourage nested carousels

```suggestion
				selector=".stk-block-carousel__dots"
```